cmake/projects/PocoCpp/hunter.cmake

```diff
@@ -8,6 +8,17 @@ hunter_add_version(
     PACKAGE_NAME
     PocoCpp
     VERSION
+    v1.7.8-p0
```

Comment:
Should be just `1.7.8-p0`, not `v1.7.8-p0`. This form used for most of versions.